app/views/womens_allocations/show.html.erb

```diff
@@ -0,0 +1 @@
+<h1>Hello</h1>
```

Comment:
oops! What is meant to be in this template?

Reply:
Oh dear